package.json

```diff
@@ -84,6 +82,7 @@
     "postinstall": "npm run dist",
     "fix": "npm run fix:js",
```

Comment:
add the `&& npm run fix:non-js`

Reply:
or https://github.com/wireapp/wire-webapp/blob/dev/package.json#L100

```
"fix": "npm-run-all fix:*",
```

Reply:
I will skip the depenceny to `npm-run-all` but thanks. :)